src/qibolab/instruments/qblox/cluster_qrm_rf.py

```diff
@@ -1075,26 +1073,21 @@ def acquire(self):
         """
         # wait until all sequencers stop
         time_out = int(self._execution_time) + 60
-
         t = time.time()
         for sequencer_number in self._used_sequencers_numbers:
             while True:
-                try:
-                    state = self.device.get_sequencer_state(sequencer_number)
-                except:
-                    pass
-                else:
-                    if state.status == "STOPPED":
-                        # log.info(f"{self.device.sequencers[sequencer_number].name} state: {state}")
-                        # TODO: check flags for errors
-                        break
-                    elif time.time() - t > time_out:
-                        log.info(
-                            f"Timeout - {self.device.sequencers[sequencer_number].name} state: {state}"
-                        )
-                        self.device.stop_sequencer(sequencer_number)
-                        break
-                time.sleep(1)
+                state = self.device.get_sequencer_state(sequencer_number)
+
+                if state.status == "STOPPED":
+                    # TODO: check flags for errors
+                    break
+                elif time.time() - t > time_out:
+                    log.info(
+                        f"Timeout - {self.device.sequencers[sequencer_number].name} state: {state}"
+                    )
+                    self.device.stop_sequencer(sequencer_number)
+                    break
+                time.sleep(0.5)
```

Comment:
Why the sleep? And how do you decide how much time?

Reply:
Actually I'm not sure, I suspect there is some sort of delay to get the state from the sequencer (?), I should ask to Alvaro but I kept this choice reducing it without any effect.